index.js

```diff
@@ -0,0 +1,13 @@
+import BookList from './module/bookList.js';
+import { DateTime } from './module/luxon.js';
+import Navigation from './module/nav-page.js';
+
+const bookObj = new BookList();
+const nav = new Navigation();
+
+const dateContainer = document.querySelector('.date-container');
+const now = DateTime.now();
+dateContainer.innerHTML = `${now.day}/${now.month}/${now.year} ${now.hour}:${now.minute}`;
+
+bookObj.loadBooks();
+window.onload(nav);
```

Comment:
- [ ] @Kwaku28 great job on your app, however, I am getting the console error as shown below, kindly fix it
![image](https://user-images.githubusercontent.com/61408860/228047179-9ebb8bb2-8e6e-4ba0-8f19-a8523d631d90.png)